RockWeb/Blocks/Crm/PersonDetail/EditGroup.ascx.cs

```diff
@@ -1246,20 +1229,20 @@ protected void btnSave_Click( object sender, EventArgs e )
                                     }
                                 }
                             }
+
```

Comment:
I'm not super familiar with this code yet and I haven't gotten to the bottom of this change yet... can you briefly describe why you moved this to happen only inside the '// existing group members' block?